04_numeronym/sksandani/main.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	fmt.Fprintln(out, numeronyms("accessibility", "Kubernetes", "abc"))
+}
+
+func numeronyms(vals ...string) []string {
+	results := make([]string, len(vals))
+	in := []string{}
+	copy(in, vals)
+	i := 0
+	for _, val := range vals {
+		results[i] = getNumeronyms(val)
+		i++
+	}
+	return results
+}
+
+func getNumeronyms(val string) string {
+	var buffer bytes.Buffer
+	in := strings.Trim(val, " ")
```

Comment:
I would also trim newlines `\n` and tabs `\t`
`strings.Trim(val, " \n\t")`
Should also use a more descriptive name than `in`.